backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -62,6 +62,7 @@ def index
       def new
         user = Spree.user_class.find_by(id: params[:user_id]) if params[:user_id]
         @order = Spree::Core::Importer::Order.import(user, order_params)
+        associate_user_address(user)
```

Comment:
Did you consider moving this operation into `Spree::Core::Importer::Order` instead? 

I think it would be "just" matter of preparing the `order_params` to also have the `ship_address_attributes` and `bill_address_attributes` fields before calling the `import` method. What do you think?

Reply:
I agree. We thought about this approach too, it's good to have some backup. I updated the PR, let me know what you think. And sorry for the long wait! 🙇